components/modal/store/StoreManualModal.tsx

```diff
@@ -157,17 +170,30 @@ export default function StoreManualModal({ radioMode }: StoreManual) {
               {item.title}
               <ul className={styles.ruleDetail}>
                 {item.description.map((rule, idx) =>
-                  rule.tag !== '' ? (
+                  rule.tag === '' ? (
+                    rule.content.map((content, idx) => (
+                      <li key={idx}>{content}</li>
+                    ))
+                  ) : rule.tag === '🔗 뽑기 확률표' ? (
                     <li key={idx}>
-                      {rule.tag}
+                      <a href='https://www.notion.so/21cadc74ddb245ea9494c7b203892c83?pvs=4'>
+                        {rule.tag}
+                      </a>
```

Comment:
지금은 링크를 눌렀을 때 현재 페이지에서 노션 페이지가 열리는데 target 속성 추가하면 새 창에서 열리게 할 수도 있어요!
필요하다면 추가해도 좋을 것 같습니당
```suggestion
                      <a href='https://www.notion.so/21cadc74ddb245ea9494c7b203892c83?pvs=4' target='_blank'>
                        {rule.tag}
                      </a>
```

Reply:
새 창에서 열리게 하는게 더 좋네요! 수정하겠습니당